awstesting/integration/performance/s3DownloadManager/main_test.go

```diff
@@ -19,78 +21,143 @@ import (
 var benchConfig BenchmarkConfig
 
 type BenchmarkConfig struct {
-	bucket       string
-	tempdir      string
-	clientConfig ClientConfig
+	bucket         string
+	tempdir        string
+	clientConfig   ClientConfig
+	sizes          string
+	parts          string
+	concurrency    string
+	bufferSize     string
+	uploadPartSize int64
 }
 
 func (b *BenchmarkConfig) SetupFlags(prefix string, flagSet *flag.FlagSet) {
 	flagSet.StringVar(&b.bucket, "bucket", "", "Bucket to use for benchmark")
 	flagSet.StringVar(&b.tempdir, "temp", os.TempDir(), "location to create temporary files")
+	flagSet.StringVar(&b.sizes, "size", fmt.Sprintf("%d,%d", 5*sdkio.MebiByte, 1*sdkio.GibiByte), "file sizes to benchmark separated by comma")
+	flagSet.StringVar(&b.parts, "part", fmt.Sprintf("%d,%d,%d", s3manager.DefaultDownloadPartSize, 25*sdkio.MebiByte, 100*sdkio.MebiByte), "part sizes to benchmark separated by comma")
```

Comment:
Nit breaking up these statements to be split between multiple lines will make it easier to read.